.github/workflows/stale.yml

```diff
@@ -16,11 +16,9 @@ jobs:
     - uses: actions/stale@v1
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'This issue had no activity for 30 days. In the absence of activity or the "do-not-close" label, the issue will be automatically closed within 7 days.'
-        stale-pr-message: 'This pull request had no activity for 30 days. In the absence of activity or the "do-not-close" label, the pull request will be automatically closed within 7 days.'
+        stale-issue-message: 'A friendly reminder that this issue had no activity for 30 days.'
+        stale-pr-message: 'A friendly reminder that this PR had no activity for 30 days.'
```

Comment:
Are we not closing these after 7 days now?  If so, I think that should still be noted.

Reply:
No, we are closing after one year. The intention is to not close at all but the docs of the stale workflow do not indicate whether we can not close, so a limit of 365 days seems fair.